### ISSUE TITLE
Add support for linux? [meson]

### ISSUE THREAD

**Issue by neutrino2211** [TheCherno/WalnutAppTemplate#1]
I've managed to get this example running on linux using the meson build system without being invasive on the part of the submodules. Is this something that can be added here?

Here's my implementation https://github.com/neutrino2211/WalnutLinuxSanityCheck

The major additions are in the scripts folder

**Comment by Nonoreve** [comment]
Its so good you provide extensive installation instructions.

**Comment by neutrino2211** [comment]
@Nonoreve Yeah. Installation instructions for anything linux are complicated, so I just copied the Vulkan instructions word for word.

**Comment by sudopluto** [comment]
please see https://github.com/TheCherno/WalnutAppTemplate/pull/4 for premake based version of linux support
